test/docfx.Test/lib/JsonSchemaTest.cs

```diff
@@ -371,5 +371,38 @@ public void TestJsonSchemaValidation(string schema, string json, string expected
             var actual = string.Join('\n', errors.Select(err => err.ToString().Replace("\\r", "")).OrderBy(err => err).ToArray()).Replace('"', '\'');
```

Comment:
consider adding a spec test in `validation.yml` to cover the integration between docfx and `JsonSchemaValidator`